cmd/fleet/serve.go

```diff
@@ -472,7 +472,7 @@ the way that the Fleet server works.
 			if _, err := startAutomationsSchedule(ctx, instanceID, ds, logger, 5*time.Minute, failingPolicySet); err != nil {
 				initFatal(err, "failed to register automations schedule")
 			}
-			if _, err := startIntegrationsSchedule(ctx, instanceID, ds, logger); err != nil {
+			if _, err := startIntegrationsSchedule(ctx, instanceID, ds, logger, license); err != nil {
```

Comment:
I wonder if we are at a point where we start talking about having something like a singleton for license and configs.

Reply:
How about I tackle this in a subsequent PR, as I think it would allow removing it from a number of places. It will also make testing a bit more painful but we can introduce a testing helper to call instead of `context.Background()`.

Reply:
Definitely, this sounds like a bigger refactor.

Reply:
https://github.com/fleetdm/fleet/issues/8454 I assigned myself and will try to address it this PM before starting on something else.